.github/workflows/e2e-test.yaml

```diff
@@ -74,6 +74,8 @@ jobs:
 
       - name: Create kind cluster
         uses: helm/kind-action@v1.2.0
+        with:
+          node_image: "kindest/node:v1.21.1"
```

Comment:
I don't think it's necessary to pass `node_image` here at all; because it appears that `v1.21.1` is being passed by default, going by this CI job:
https://github.com/grafana/k6-operator/actions/runs/6512291825/job/17689689080

The real question is how to pass `kindest/node:v1.27.1`? By your comment, does it mean that `kind-action` must of another version?

Reply:
Actually, I've just realized that in my deps update PR, for non-Helm e2e test I needed to change kind version together with node:
https://github.com/grafana/k6-operator/blob/30577947ac70e51717e5e1ca4b1c84c84dceb11d/.github/workflows/e2e-test.yaml#L46
:thinking:  Maybe it's the same in Helm?

Reply:
I confused that the commit [3057794](https://github.com/grafana/k6-operator/commit/30577947ac70e51717e5e1ca4b1c84c84dceb11d) is HEAD now isn't it?
https://github.com/grafana/k6-operator/blob/30577947ac70e51717e5e1ca4b1c84c84dceb11d/.github/workflows/e2e-test.yaml#L79
Would you like to use kind v0.20.0?
We should upgrade `helm/kind-action` because `helm/kind-action@v1.2.0` used the old kind(v0.11.1) version by default following this document.

https://github.com/helm/kind-action/releases/tag/v1.2.0

Reply:
@u5surf I've just been in the middle of testing the branch with that commit (PR #308 ) and merged it in just now. Sorry for confusion! So you'll need to do `git rebase` on main (it'll also remove failing unit tests from your PR).

> Would you like to use kind v0.20.0?
We should upgrade helm/kind-action because helm/kind-action@v1.2.0 used the old kind(v0.11.1) version by default following this document.

Thanks for checking! Yes, I think let's go with `kind-action` upgrade then and 1.27.1 node image. Hopefully, it works!